run/testing/jobs.e2e_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudruntests
+
+import (
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/cloudrunci"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestCloudRunJobs(t *testing.T) {
+	tc := testutil.EndToEndTest(t)
+
+	crj := &cloudrunci.Job{
+		Name:        "runjobs",
+		ProjectID:   tc.ProjectID,
+		Dir:         "../jobs",
```

Comment:
out-of-scope: I missed before that the Cloud Run Jobs quickstart is directly in run/jobs. It should be in run/jobs/quickstart or similar, one more directory down, so we have clear separation in place to add more sample jobs.